web/src/layouts/layoutPage.vue

```diff
@@ -17,6 +17,27 @@
 -->
 
 <script setup lang="ts">
+import type { MenuOption } from 'naive-ui';
+import SiderMenu from './components/menus/index.vue'
+import type { RouteRecordRaw } from 'vue-router';
+import { children_routes } from '@/router/routes';
+
+const menus = ref<MenuOption[]>([])
+
+const mapRouterToMenu = (routes:RouteRecordRaw[]) => {
+  if (routes) {
+    menus.value = routes.map(item => {
+      return {
+        label: item.meta?.title ?? '',
+        key: item.path,
+        path:item.path,
```

Comment:
Missing space

Reply:
Could this format with `eslint --fix` via eslint-plugin-vue?

Reply:
oh~ I'm sorry, I ran the `eslint` script. But I forgot to run the `format` script.